hack/test

```diff
@@ -68,8 +62,10 @@ case $buildmode in
   ;;
 esac
 
+cacheVolume=$(docker create -v /root/.cache -v /root/.cache/registry alpine)
```

Comment:
I'm not huge fan of contaminating /root/.cache on the host.
Does `docker volume create` work?